src/StatusPageEmbed.js

```diff
@@ -0,0 +1,281 @@
+/* eslint-env browser */
+import React, { Component } from 'react';
+import { string, arrayOf, number, oneOf, bool } from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes, faInfoCircle, faExclamationTriangle } from '@fortawesome/pro-light-svg-icons';
+
+import 'abortcontroller-polyfill/dist/abortcontroller-polyfill-only';
+import {fetch as fetchPolyfill} from 'whatwg-fetch';
+
+import './StatusPageEmbed.scss';
+
+// use native browser implementation if it supports aborting
+const fetch = ('signal' in new Request('')) ? window.fetch : fetchPolyfill;
+
+const abortController = new AbortController();
+const initialState = {
+  status: {
+    indicator: 'none',
+    description: 'All Systems Operational'

```

Comment:
I feel like it'd be sensible to pull the 'standard' descriptors (ASO, PDS) into constants, either in this class or externally. Slightly magic as it stands.